### ISSUE TITLE
Extend the explanation when adding an expression profile

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/PK-Sim#2371]
![grafik](https://user-images.githubusercontent.com/25061876/198254498-ca63cf9c-b4e6-43f9-8eef-bfe711c7620f.png)

Add another sentence, something like "If Gene Expression DB for the selected species is configured, it can be used in the next step"

**Comment by StephanSchaller** [comment]
Yes, in the workshop we just had are having, it was unclear to users that a second step is needed. Also, it was unclear why a preselection list of enzymes is available in the first step.
And we really need to label if repopulate values for Ref. Conc and half lives are pre-informed or not (maybe with "Value Origin"?)
